css/styles.css

```diff
@@ -0,0 +1,17 @@
+html,
+body {
+    margin: : 0;
+    padding: 0
+}
+.product {
+    border: 1px solid black;
+    width: 15%
+}
+.cart {
+    border: 1px solid black;
+    height: 30%
+}
+.bin {
```

Comment:
As it is a cart application , the bin provided to remove some products from cart to bin. That is the reason why name 'bin' is used.